jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageReference.java

```diff
@@ -134,9 +144,12 @@ public static ImageReference parse(String reference) throws InvalidImageReferenc
   }
 
   /**
-   * @param registry the image registry
+   * Constructs an {@link ImageReference} from the image reference components, consisting of an
+   * optional registry, a repository, and an optional tag.
+   *
+   * @param registry the image registry, or {@code null} to use the default registry (Docker Hub)
    * @param repository the image repository
-   * @param tag the image tag
+   * @param tag the image tag, or {@code null} to ues the default tag ({@code latest})
```

Comment:
_to ~~ues~~ use_
`{@link #DEFAULT_TAG}`?

Reply:
I was thinking since `latest` is a image reference spec constant, it should be referred to exactly, whereas `DEFAULT_TAG` is just a Jib constant that should always be equal to the real spec constant.

Reply:
Makes sense.

Reply:
"to ues" --> "to use"

Reply:
> I was thinking since `latest` is a image reference spec constant, it should be referred to exactly, whereas `DEFAULT_TAG` is just a Jib constant that should always be equal to the real spec constant.

Are you saying `DEFAULT_TAG` may not be `latest`?

Reply:
`DEFAULT_TAG` will always be `latest`, so having a link will just add an extra jump for readers of the javadoc.